lib/puppet/pops/parser/egrammar.ra

```diff
@@ -581,6 +581,7 @@ node_definition_expression
 #   TODO: The error that "class" is not a valid classname is bad - classname rule is also used for other things
 classname
   : NAME
+  | WORD
```

Comment:
The grammar was changed, but not regenerated. Did you forget to check it in? (Alternatively, if you did not regenerate the parser, this change was not needed since it only takes effect after the parser has been regenerated).